addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -229,7 +226,7 @@ registerModel({
                     this.update({ isPreparingLoading: true });
                     this.async(() =>
                         new Promise(resolve => {
-                            this._loaderTimeout = this.messaging.browser.setTimeout(resolve, 400);
+                            this.update({ loaderTimeout: this.messaging.browser.setTimeout(resolve, this.messaging.loadingBaseDelayDuration) });
```

Comment:
This looks like it is not covered by a test, maybe add a test some day.

Reply:
Also it's a bit ugly with that small grey bar, I'm not sure if it was always like, or if many changes over the years made it different?

![image](https://user-images.githubusercontent.com/38213504/155003528-84131482-429d-489c-926f-522dc4fd8590.png)